apps/docs/src/app/core/component-docs/list/examples/list-data-example/list-data-example.component.html

```diff
@@ -12,16 +13,28 @@
         (addOnButtonClicked)="handleSearchTermChange('')"
     >
     </fd-input-group>
-    <button fd-button compact="true" glyph="sort-ascending" (click)="changeSort(true)"></button>
-    <button fd-button compact="true" glyph="sort-descending" (click)="changeSort(false)"></button>
+    <button
+        ariaLabel="Sort ascending"
+        fd-button
+        compact="true"
+        glyph="sort-ascending"
+        (click)="changeSort(true)"
+    ></button>
+    <button
+        ariaLabel="Sort descending"
+        fd-button
+        compact="true"
+        glyph="sort-descending"
+        (click)="changeSort(false)"
+    ></button>
 </fd-toolbar>
 <ul fd-list>
     <li fd-list-item *ngFor="let item of displayedItems | sortBy: ascendingSort; let ind = index">
         <span fd-list-title>
             {{ item }}
         </span>
         <span fd-list-secondary>
-            <button fd-button compact="true" glyph="delete" (click)="removeItem(ind)"></button>
+            <button ariaLabel="Remove" fd-button compact="true" glyph="delete" (click)="removeItem(ind)"></button>
```

Comment:
this is a native html element, a button. Why don't you use `aria-label`? Why did you decide go for an input property?

Reply:
There's a host binding in `fd-button` for aria-label, that can override plain `aria-label` attribute